### PR TITLE
Adding Bp and BPA resources with examples to backupdr product

### PR DESCRIPTION
**Pull request by niharika-98**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
Adding BP and BPA resource to Backupdr product
```

### CODE REVIEW

File: mmv1/products/backupdr/BackupPlan.yaml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'BackupPlan'
+# Remove this towards GA. b/371938375
```
Comment (gTheSattiraju) [inline mmv1/products/backupdr/BackupPlan.yaml:16]: This is probably not needed anymore

File: mmv1/products/backupdr/BackupPlan.yaml [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+delete_url: projects/{{project}}/locations/{{location}}/backupPlans/{{backup_plan_id}}
+description: A backup plan defines when and how to back up a resource, including the backup's schedule, retention, and location.
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/backup-disaster-recovery/docs'
+  api: 'https://cloud.google.com/backup-disaster-recovery/docs/reference/rest'
+autogen_async: true
+timeouts:
+  insert_minutes: 20
+  delete_minutes: 20
```
Comment (gTheSattiraju) [inline mmv1/products/backupdr/BackupPlan.yaml:31]: Are these standard API timeouts?

File: mmv1/products/backupdr/BackupPlan.yaml [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+examples:
+  - min_version: beta
+    name: 'backup_dr_backup_plan'
+    primary_resource_id: 'my-backup-plan'
```
Comment (gTheSattiraju) [inline mmv1/products/backupdr/BackupPlan.yaml:35]: Isn't this same as the name? 🤔

File: mmv1/products/backupdr/BackupPlan.yaml [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+description: A backup plan defines when and how to back up a resource, including the backup's schedule, retention, and location.
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/backup-disaster-recovery/docs'
+  api: 'https://cloud.google.com/backup-disaster-recovery/docs/reference/rest'
+autogen_async: true
+timeouts:
+  insert_minutes: 20
+  delete_minutes: 20
+examples:
```
Comment (gTheSattiraju) [inline mmv1/products/backupdr/BackupPlan.yaml:32]: Could you move examples to the bottom of the file?

File: mmv1/products/backupdr/BackupPlanAssociation.yaml [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+description: A Backup and DR BackupPlanAssociation.
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/backup-disaster-recovery/docs'
+  api: 'https://cloud.google.com/backup-disaster-recovery/docs/reference/rest'
+autogen_async: true
+timeouts:
+  insert_minutes: 20
+  delete_minutes: 20
+examples:
```
Comment (gTheSattiraju) [inline mmv1/products/backupdr/BackupPlanAssociation.yaml:32]: Move this to the bottom of the file

File: mmv1/products/backupdr/BackupPlanAssociation.yaml [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+      backup_plan_name: 'my-backup-plan'
+      backup_rule_name: 'rule-1'
+      backup_retention_days: 5
+      recurrence_type: 'HOURLY'
+      hourly_frequency: 6
+      time_zone: "UTC"
+      start_hour_of_day: 5
+      end_hour_of_day: 20
+      backup_plan_association_name: 'bpa1'
+      resource_name: 'projects/gcp-project/locations/us-central1/{instance_id}'
```
Comment (gTheSattiraju) [inline mmv1/products/backupdr/BackupPlanAssociation.yaml:49]: This list seems to show the output_only fields as well. You might want to revisit this.

File: mmv1/products/backupdr/BackupPlan.yaml (hunk 4 above)
Comment (niharika-98) [reply mmv1/products/backupdr/BackupPlan.yaml:30]: AS per convention they stay at top. Would you still want me to move it to bottom?

File: mmv1/products/backupdr/BackupPlan.yaml (hunk 3 above)
Comment (niharika-98) [reply mmv1/products/backupdr/BackupPlan.yaml:33]: No, primary_resource_id is tf resource name while name is example name

File: mmv1/products/backupdr/BackupPlanAssociation.yaml (hunk 5 above)
Comment (niharika-98) [reply mmv1/products/backupdr/BackupPlanAssociation.yaml:30]: This is as per convention

File: mmv1/products/backupdr/BackupPlan.yaml (hunk 1 above)
Comment (niharika-98) [reply mmv1/products/backupdr/BackupPlan.yaml]: Removed it

File: mmv1/products/backupdr/BackupPlan.yaml (hunk 2 above)
Comment (niharika-98) [reply mmv1/products/backupdr/BackupPlan.yaml]: Updated standard tineouts,

File: mmv1/products/backupdr/BackupPlanAssociation.yaml (hunk 6 above)
Comment (niharika-98) [reply mmv1/products/backupdr/BackupPlanAssociation.yaml]: fixed by adding field values in example file onkly

**Review by gTheSattiraju — APPROVED** [review]

**Pull request merged**